#--- webkitgtk-2.38.5/Source/WebKit/PlatformGTK.cmake.orig	2022-09-14 13:58:10.556863500 +0200
#+++ webkitgtk-2.38.5/Source/WebKit/PlatformGTK.cmake	2023-04-22 19:21:53.186764446 +0200
#@@ -764,3 +764,6 @@ GI_INTROSPECT(WebKit2WebExtension ${WEBK
# GI_DOCGEN(WebKit2WebExtension gtk/webkit2gtk-webextension.toml.in
#     CONTENT_TEMPLATES gtk/urlmap.js
# )
#+
#+add_dependencies(gir-WebKit2 gir-JavaScriptCore)
#+add_dependencies(gir-WebKit2WebExtension gir-JavaScriptCore)
--- webkitgtk-2.38.5/Source/cmake/FindGI.cmake.orig	2023-02-14 10:01:07.457508800 +0100
+++ webkitgtk-2.38.5/Source/cmake/FindGI.cmake	2023-04-22 19:21:53.186764446 +0200
@@ -265,7 +265,7 @@ function(GI_INTROSPECT namespace nsversi
             get_property(dep_gir_lib TARGET "gir-${dep}" PROPERTY GI_GIR_LIBRARY)
             if (dep_gir_path)
                 list(APPEND scanner_flags "--include-uninstalled=${dep_gir_path}")
-                list(APPEND gir_deps "${dep_gir_path}")
+                list(APPEND gir_deps "gir-${dep}")
             else ()
                 message(AUTHOR_WARNING
                     "Target '${dep}' listed as a dependency but it has not "
